pwndbg/heap/ptmalloc.py

```diff
@@ -662,7 +658,18 @@ def initialize_structs(method):
     def wrapper(self, *args, **kwargs):
         # If structs module for HeuristicHeap is None, try to initialize it.
         if self._structs_module is None:
-            self._structs_module = importlib.reload(importlib.import_module('pwndbg.heap.structs'))
+            try:
+                self._structs_module = importlib.reload(importlib.import_module('pwndbg.heap.structs'))
+            except TypeError as e:
+                print(
+                    message.hint(
+                        "There is a TypeError when importing the structs for heap heuristic.\n"
+                        "Maybe it's because we can't find the GLIBC version\n"
+                        "Try `set glibc <version>` to set it manually (e.g. set glibc 2.31 )\n"
+                        "Use `heap_config` to show the current config about heap heuristics.\n"
+                    )
+                )
+                raise e
```

Comment:
Shall we really raise further?

Reply:
Maybe we don't need it, I did that before because I want to make sure the command stops here, but it seems that there must have some error that can stop the command immediately after we can't import it.
But maybe we raise it here will let the debug process more easily and can be detectable more easily?